### PR TITLE
Changed how the default database URL is read

### LINKED ISSUE #804: Read from `KC_DATABASE_URL`
and fall back to `DATABASE_URL` if it's not present. This should be a few lines' change in a settings file.

See https://github.com/kobotoolbox/kobo-docker/issues/264#issuecomment-1084033452. KPI counterpart: https://github.com/kobotoolbox/kpi/issues/3748

### PR DESCRIPTION
**Pull request by JacquelineMorrissette**
## Description
Changed the default database to read from `KC_DATABASE_URL` and fallback to `DATABASE_URL`

## Related Issues
closes #804

### CODE REVIEW

File: onadata/settings/base.py [hunk 1]
```diff
@@ -351,9 +351,12 @@ def skip_suspicious_operations(record):
 DEBUG = env.bool('DJANGO_DEBUG', True)
 
 # Database (i.e. PostgreSQL)
-DATABASES = {
-    'default': env.db(default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)
-}
+DATABASES = {}
+if 'KPI_DATABASE_URL' in env.ENVIRON:
```
Comment (noliveleger) [inline onadata/settings/base.py:355]: You meant `KC_DATABASE_URL`?  😄

File: onadata/settings/base.py [hunk 2]
```diff
@@ -351,9 +351,12 @@ def skip_suspicious_operations(record):
...
+    DATABASES['default'] = env.db_url(default='KC_DATABASE_URL')
```
Comment (noliveleger) [inline onadata/settings/base.py:356]: I think the syntax should be `env.db_url('KC_DATABASE_URL', default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)`

Comment (noliveleger) [inline onadata/settings/base.py:356]: I think the syntax would be `env.db_url('KC_DATABASE_URL', default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)`

File: onadata/settings/base.py [hunk 3]
```diff
@@ -351,9 +351,12 @@ def skip_suspicious_operations(record):
...
+else:
+    DATABASES['default'] = env.db_url(default='DATABASE_URL')
```
Comment (noliveleger) [inline onadata/settings/base.py:358]: I think the syntax should be `env.db_url('DATABASE_URL', default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)`.

Comment (noliveleger) [inline onadata/settings/base.py:358]: I think the syntax should be `env.db_url('DATABASE_URL', default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)`.

**Review by noliveleger — CHANGES_REQUESTED** [review]

**Comment by jnm** [comment]
I could use this in my local development environment right now :grin:

I think the "fall back" part may be absent from the latest revisions?

> Read from `KC_DATABASE_URL` and fall back to DATABASE_URL if it's not present.

**Comment by noliveleger** [comment]
> I could use this in my local development environment right now 😁
> 
> I think the "fall back" part may be absent from the latest revisions? From the description of #804:
> 
> > Read from `KC_DATABASE_URL` and fall back to DATABASE_URL if it's not present.

Indeed.

**Review by jnm — APPROVED** [review]

**Pull request merged**